extensions/virtuallistview/src/VirtualListView.tsx

```diff
@@ -63,14 +63,21 @@ export interface VirtualListViewItemInfo extends VirtualListCellInfo {
     isNavigable?: boolean;
 }
 
+export interface VirtualListViewCellRenderDetails<T extends VirtualListViewItemInfo> extends VirtualListCellRenderDetails<T> {
+
+}
+
 export interface VirtualListViewProps<ItemInfo extends VirtualListViewItemInfo> extends RX.CommonStyledProps<RX.Types.ViewStyleRuleSet> {
     testId?: string;
 
     // Ordered list of descriptors for items to display in the list.
     itemList: ItemInfo[];
 
     // Callback for rendering item when it becomes visible within view port.
-    renderItem: (item: ItemInfo, hasFocus: boolean) => JSX.Element | JSX.Element[];
+    renderItem: (renderDetails: VirtualListCellRenderDetails<ItemInfo>) => JSX.Element | JSX.Element[];
+    onItemSelected?: (item: ItemInfo) => void;
+
+    initialSelectedKey?: string;
```

Comment:
I'm not a fan of props that have different semantics based on whether the component was just mounted or not. There are a few other examples of this in RX (notably, the `defaultValue` prop in RX.TextInput), and they tend to cause problems and require the caller to use a `key` to force a mount/unmount in certain cases. I think it would be better to provide a programmatic accessor to set the current selection, and users could call that explicitly after mounting is complete if they want to set the initial selection.

Reply:
If you don't provide this key, the VLV will automatically make the wrong element the initially selected/focused item. Calling a function to set this would result in an additional render cycle of 2 VLV items, which isn't the end of the world but in some cases could be expensive if the components are heavy.  This is optional, so consumers could also do what you're suggesting

Reply:
OK, I'm still not a fan of these types of props, but you make a good argument. I'm fine leaving it as is.